src/HAL/nanoHAL_StorageOperation.cpp

```
@@ -0,0 +1,151 @@
+#include <nanoHAL_v2.h>
+#include <nanoWeak.h>
+#include <nanoCLR_Runtime.h>
+#include <nanoCLR_Types.h>
+#include <nanoPAL_FileSystem.h>
+#include <nanoCLR_FileStream.h>
+#include <nanoHAL_StorageOperation.h>
+#include <WireProtocol_MonitorCommands.h>
+
+#if CONFIG_NF_FEATURE_HAS_ACCESSIBLE_STORAGE
+
+uint32_t HAL_StorageOperation(uint8_t operation, uint32_t dataLength, uint32_t offset, uint8_t *data)
```

Cppcheck (2.21.0)

[style] The function 'HAL_StorageOperation' is never used.

(unusedFunction)